docker-compose.env.example

```diff
@@ -1,4 +1,4 @@
-DJANGO_SECRET_KEY=
+DJANGO_SECRET_KEY=&_hnt6o2phv^vrr(%!4wo#wh2fh!0t-7=3!)8tr4so@y#d37yy

```

Comment:
Should this not be `SECRET_KEY_GOES_HERE` so that scripts/setup works

Reply:
Ah, good spot 👍